uint16 robot_id
uint16 cluster_id
uint16 instance_number
uint16 iteration_number
bool is_auxiliary
uint32[] pose_ids
dpgo_ros/MatrixMsg[] poses